app/models/custom_field.rb

```diff
@@ -76,7 +76,7 @@ class CustomField < Ekylibre::Record::Base
   end
 
   validate do
-    unless customized_type.to_s.constantize.respond_to?(:custom_fields)
+    unless customized_type.nil? || customized_type.to_s.constantize.respond_to?(:custom_fields)
```

Comment:
For clearness purpose, can we rewrite condition like: `if customized_type.present? && !custom...` 
It 's seems to me more understandable